packages/ContentsGroupingExtensions/src/components/ChildRelationField/index.vue

```diff
@@ -1,32 +1,49 @@
 <template>
     <div>
```

Comment:
I guess no need to wrap `<div>`, just use `<ExtRelationFld v-if="flagToLoadOnce" v-bind="..." />` on root level.